docs/plugins/npe2_getting_started.md

```diff
@@ -194,13 +194,14 @@ napari.manifest =
 
 The manifest file is specified relative to the submodule root path.
 So for the example it will be loaded from:
-`<path/to/npe2-tester>/napari.yaml`.
+`<path/to/npe2-tester>/npe2_tester/napari.yaml`.
```

Comment:
should this be `.yml` too?

Reply:
You're right, it's `.yml` right now. I'd rather change it to `.yaml` on the cookiecutter side though (napari/cookiecutter-napari-plugin#76).

Reply:
ok that's fine. i guess this can be merge now then